README.md

```diff
@@ -45,39 +45,30 @@ Now, prepare to be amazed by the wonders of binaries.
 
 ## Usage
 
-Replace `require(zmq)` in your code base with `zmq-prebuilt`. That's it.
+Replace `require(zmq)` in your code base with `require(zmq-prebuilt)`. That's it.
 The wonder of binaries begins.
 
 ----
 
 ## Installation - Contributors and Development
 
 To set up `zmq-prebuilt` for development, fork this repository and
-clone your fork to your system. Be sure you have `git-lfs` installed.
+clone your fork to your system. Be sure you have Python 2 and `git-lfs` installed.
 
-### Linux and OS X
-
-*Prerequisites*
+**Prerequisites for Linux and OS X**
 
 If you are running on Linux or OS X, you will need to have `automake`,
-`autoconf`, `wget` and `libtool` installed. For Linux, use your distribution's
+`autoconf`, `pkg-config`, `wget` and `libtool` installed. For Linux, use your distribution's
 package manager to install. On OS X, these can be installed using
 [Homebrew](http://brew.sh) and using the Homebrew command `brew install`
 command. For example, install `wget` with `brew install wget`.
 
-Install a development version of `zmq-prebuilt` with the following:
-
-```bash
-./build_libzmq.sh
-npm install
-```
+**Prerequisites for Windows**
 
-### Windows
+On Windows you'll need [Visual Studio 2013](https://www.microsoft.com/en-US/download/details.aspx?id=44914).
```

Comment:
Appveyor has almost every version installed. [This line](https://github.com/nteract/zmq-prebuilt/blob/master/binding.gyp#L11) makes sure that gyp uses  Visual Studio 2013.

I tried it on a system where only Visual Studio 2015 was installed and gyp complained.